charts/pega/templates/_pega-credentials-secret.tpl

```diff
@@ -4,6 +4,10 @@ apiVersion: v1
 metadata:
   name: {{ template "pegaCredentialsSecret" $ }}
   namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-delete
```

Comment:
I think we can still have `post-install` and `post-upgrade`. But can we evaluate the necessity of having `pre-delete` here and which one is more efficient?